### PR TITLE
Cmake

### PR DESCRIPTION
**Pull request by IliaIlyin**
Task DTO changed - now it does not have subtasks field.

### CODE REVIEW

File: src/API/TaskService.h [hunk 1]
```diff
@@ -23,7 +23,8 @@ class TaskService {
                                   Task::Priority priority = Task::Priority::NONE, std::string label = "");
   bool                       addTask(Task &task);
   bool                       addSubTaskToParent(TaskDTO &parent, Task &task);
-  std::optional<TaskDTO>     getTask(TaskDTO &task_entity);
+  std::optional<TaskDTO>     getTask(TaskID id);
+  std::optional<std::vector<TaskDTO>> getSubtasks(TaskID id);
```
Comment (IlliaDmitriiev) [inline src/API/TaskService.h:27]: i'm not sure that you need optional for vector

Comment (IliaIlyin) [reply src/API/TaskService.h:27]: it's better to distinguish "task corresponding to id is not found" and "task with such id does not have subtasks"

Comment (IlliaDmitriiev) [reply src/API/TaskService.h:27]: ok

**Comment by IliaIlyin** [comment]
accidentally i merged the request

**Pull request merged**